test/fixtures/permission/fs-traversal.js

```diff
@@ -65,35 +65,15 @@ const uint8ArrayTraversalPath = new TextEncoder().encode(traversalPath);
   fs.readFile(bufferTraversalPath, common.expectsError({
     code: 'ERR_ACCESS_DENIED',
     permission: 'FileSystemRead',
-    resource: traversalPath,
+    resource: path.toNamespacedPath(traversalPath),
   }));
 }
 
 {
   fs.readFile(uint8ArrayTraversalPath, common.expectsError({
     code: 'ERR_ACCESS_DENIED',
     permission: 'FileSystemRead',
-    resource: traversalPath,
-  }));
-}
-
-// Monkey-patching path module should also not allow path traversal.
```

Comment:
Regardless if we are moving the implementation to C++ I believe `path.toNamespacedPath` will always exist through an API right? If so, I don't think we should remove this test, it won't fail anyway.

Reply:
I think its not needed but on top of that It fails because win32 path resolves return "." for cwd, but previously it didn't.